dataflow/src/main/java/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -447,12 +447,13 @@ private static Receiver internalReprOfMemberSelect(
     }
 
     /**
-     * This class represents a Java expression and its type. However, it does not represent all
-     * possible Java expressions (for example, it does not represent a ternary expression; use
-     * {@link FlowExpressions.Unknown} for unrepresentable expressions). Furthermore, it supports
-     * some <a
+     * This class represents a Java expression and its type. It does not represent all possible Java
+     * expressions (for example, it does not represent a ternary expression; use {@link
+     * FlowExpressions.Unknown} for unrepresentable expressions). Furthermore, its string syntax
+     * supports some <a
      * href="https://checkerframework.org/manual/#java-expressions-as-arguments">extensions</a>,
-     * notably {@code <self>} and {@code #1} for formal parameters.
+     * notably {@code <self>} and {@code #1} for formal parameters. (There are not special
```

Comment:
From @smillst comment https://github.com/typetools/checker-framework/pull/3473#discussion_r454512286 it sounds like `<self>` isn't handled here at all. So we probably should instead highlight such checker-specific extensions?

Reply:
I will remove the mention from here, since it is specific to Checker Framework syntax.